src/components/Controls/ArrayInput/index.tsx

```diff
@@ -0,0 +1,56 @@
+import {FunctionComponent} from "react"
+import CloseIcon from "../../../assets/icons/CloseIcon"
+import Input from "../Input"
+import Button from "../Button"
+import useArrayInput from "./hooks"
+import "./styles.scss"
+
+export type ArrayInputChangeListener = (newValue: ArrayInputValue) => void
+export type ArrayInputValue = string[]
+
+const ArrayInput: FunctionComponent<{
+	onChange: ArrayInputChangeListener
+	onRemove: (indexToRemove: number) => void
+	value: string[]
+	validation?: string | null
+	borders?: "all" | "bottom"
+	placeholder?: string
+}> = ({
+	onChange,
+	onRemove,
+	value,
+	borders,
+	validation,
+	placeholder = "Type value and press enter or tab..."
+}) => {
+	const {handleInputChange, handleKeyDown, inputValue} = useArrayInput({
```

Comment:
And I don't think we ever need this hook. It's not reusable and used only to organize the logic (which is good, in general), however, this is too small component and there's of no use to move out 2 functions.